apps/cloud_composer/include/pcl/apps/cloud_composer/items/cloud_composer_item.h

```diff
@@ -111,15 +111,15 @@ namespace pcl
     //    CloudPtrT
     //    getCloudPtr () const;
         
-        /** \brief Paint View function - reimpliment in item subclass if it can be displayed in PCLVisualizer*/
+        /** \brief Paint View function - reimpiement in item subclass if it can be displayed in PCLVisualizer*/
```

Comment:
This doesn't look right (more instances)

Reply:
Fixed in [b5e1531](https://github.com/PointCloudLibrary/pcl/pull/5523/commits/b5e15319b9ed23468ffdf2792940f61cfd675879)